# Generated by Django 5.1.2 on 2024-10-23 16:22

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='bookfaultmodel',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('SDCA', models.CharField(choices=[('1', 'Select your SDCA'), ('Ahilyanagar', 'Ahilyanagar'), ('Akole', 'Akole'), ('Jamkhed', 'Jamkhed'), ('Karjat', 'Karjat'), ('Kopergaon', 'Kopergaon'), ('Nevasa', 'Nevasa'), ('Parner', 'Parner'), ('Pathardi', 'Pathardi'), ('Rahuri', 'Rahuri'), ('Rahata', 'Rahata'), ('Sangamner', 'Sangamner'), ('Shevgaon', 'Shevgaon'), ('Shrirampur', 'Shrirampur'), ('Shrigonda', 'Shrigonda')], default='1', max_length=50)),
                ('Routename', models.CharField(max_length=100)),
                ('FaultType', models.CharField(choices=[('1', 'Select the type of fault'), ('SYSTEM', 'SYSTEM'), ('OFC CUT', 'OFC CUT'), ('PATCH CHORD', 'PATCH CHORD')], default='1', max_length=100)),
                ('Reporting_date_time', models.DateTimeField()),
                ('Traffic_Affected', models.TextField()),
                ('Remarks', models.TextField(blank=True, null=True)),
                ('Fault_Id', models.IntegerField(default='')),
                ('Fault_Restored_Date_Time', models.DateTimeField(default='')),
                ('SJC_Used', models.IntegerField(default='')),
                ('OFC_Used', models.IntegerField(default='')),
                ('OFC_Type', models.CharField(default='', max_length=100)),
                ('PLB_Used', models.IntegerField(default='')),
                ('Trial_Pit', models.IntegerField(default='')),
                ('Trench', models.CharField(default='', max_length=100)),
                ('Reason_Of_Fault', models.CharField(default='', max_length=100)),
            ],
        ),
    ]
